crates/pipeline-manager/Cargo.toml

```diff
@@ -96,7 +96,7 @@ nix = { workspace = true, features = ["signal"] }  # For: terminating process gr
 # Keep in-sync with `src/compiler/rust_compiler.rs`.
 compare = { workspace = true }
 [target.'cfg(not(target_env = "msvc"))'.dependencies]
-tikv-jemallocator = { version = "0.5.4", features = ["profiling", "unprefixed_malloc_on_supported_platforms"] }
+tikv-jemallocator = { workspace = true, features = ["profiling", "unprefixed_malloc_on_supported_platforms"] }
```

Comment:
this string appears in many more places, including cargo files generated by the compiler

Reply:
good point